gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_0/02_federated_apikeys.yml

```diff
@@ -2,8 +2,9 @@ databaseChangeLog:
   - changeSet:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.